RacingCar/InputChecker.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  InputChecker.swift
+//  RacingCar
+//
+//  Created by nylah.j on 2022/04/30.
+//
+
+import Foundation
+
+struct InputChecker {
+    static func isValid(carCount: Int?) -> Bool {
```

Comment:
```suggestion
    static func isValidCarCount(_ count: Int?) -> Bool {
```
아래 isValid라는 메서드 명이 중복되어서(물론 파라미터로 구분이 가능할테지만), 이런식으로 명확하게 나눠주는건 어떨까요? 👀